src/sage/combinat/integer_vectors_mod_permgroup.py

```diff
@@ -931,8 +936,9 @@ def cardinality(self):
             # The 1 can be placed in any orbit, and by symmetry
             # it will be on the first element of the orbit.
             return Integer(len(G.orbits()))
-        if d is not None and self.has_trivial_group() and m >= d:
-            # Simple calculation with stars and bars.
+        if d is not None and m >= d and all(g.is_one() for g in G.gens()):
+            # Trivial group (should use G.is_trivial() as soon as
```

Comment:
You can and should make the pr for ·is_trivial· a dependency, then we don't need to change things again.

Reply:
Right, but I need to find out how to make a dependency between PR's (never did that before).

Reply:
There is a heading "Dependencies" at the very top of this PR, and there you simply write `#36873`.  Then, locally, you checkout the other PR (eg., `gh pr checkout 36873`, switch back to your PR, do `git merge perm_gps/is_trivial`, and push the result.

Reply:
Thanks, I hope I did it right.